product_docs/docs/biganimal/release/getting_started/01_preparing_azure/01_understanding_qotas_in_azure.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Understanding requirements in Azure"
+
+---
+
+By default, Azure sets low resource limits on the number of vCPUs (cores), and Public IP addresses permitted in your subscription. The following sections explain BigAnimal's requirements and recommended resource limits in Azure.
+
+### Public IP addresses limits
+
+A Public IP address is an IP address that used to access your cluster directly over the internet. The number of Public IP addresses must not exceed the limit for a particular region.
+
+##### Default Azure limit
+
+The default Public IP addresses limits for **Basic** and **Standard** type are set to 10. See [Public IP address limits](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#publicip-address) for more information.
+
+##### Recommended limit
+
+EDB recommends you to increase the Public IP addresses limits for **Basic** and **Standard** type to the number of clusters you plan to deploy in a given region.
+
+### vCPU limits
+
+Any time a new VM is deployed in Azure, the vCPUs for the VMs must not exceed the vCPU limits for the VM size family (see [SKU restrictions](#SKU-restrictions)) or the total regional vCPU limit. Clusters deployed in the region use separate Esv3 virtual machine cores. 
+
+The number of cores depends on the *Instance Type* and *High Availability* options of the clusters. You can calculate the number of Esv3 cores required for your cluster based on the following:
+
+* A Virtual machine instance of type E{N}sv3 uses {N} cores. For example, an instance of type E64sv3 uses 64 Esv3 cores.
+* A cluster running on an E{N}sv3 instance with high availability not enabled uses exactly {N} Esv3 cores.
+* A cluster running on an E{N}Sv3 instance with high availability enabled uses 3 * {N} Esv3 cores.
+
+For example, if you provision the largest virtual machine E64sv3 with high availability enabled, it requires (3 * 64) = 192 Esv3 cores per region. 
+
+BigAnimal uses eight Dv4 virtual machine cores to manage your BigAnimal infrastructure in a particular region. BigAnimal requires an additional eight Dv4 virtual machine cores per region.
+
+### VM size restrictions
+
+When you deploy your clusters in HA mode, BigAnimal uses multiple availability zones in the region where you plan to deploy the cluster. You might encounter en error while deploying your clusters if the VM size SKU is not available for the location of your cluster.
```

Comment:
This is conflating SKU restrictions with quotas.

Reply:
Does this make it better? :

### vCPU limits

Any time a new VM is deployed in Azure, the vCPUs for the VMs must not exceed the total vCPU limits for the region. 

Clusters deployed in the region use Esv3 virtual machine cores. The number of cores depends on the *Instance Type* and *High Availability (HA)* options of the clusters. You can calculate the number of Esv3 cores required for your cluster based on the following:

* A virtual machine instance of type E{N}sv3 uses {N} cores. For example, an instance of type E64sv3 uses 64 Esv3 cores.
* A cluster running on an E{N}sv3 instance with HA not enabled uses exactly {N} Esv3 cores.
* A cluster running on an E{N}sv3 instance with HA enabled uses 3 * {N} Esv3 cores.

For example, if you provision the largest virtual machine E64sv3 with high availability enabled, it requires (3 * 64) = 192 Esv3 cores per region. 

BigAnimal requires an additional eight Dv4 virtual machine cores per region.

##### Default limit

The default number of total vCPU (cores) per subscription per region is 20. See [Virtual Machines limits - Azure Resource Manager](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#virtual-machines-limits---azure-resource-manager) for more information.

##### Recommended limits

BigAnimal recommends the following when requesting virtual machine resource limit increases:
* Total Regional vCPUs: minimum of 60 per designated region

### Virtual machine SKU restrictions

You might encounter an error while deploying your clusters if the SKU limit for the virtual machine family is reached.

##### Recommended limits

BigAnimal recommends the following when requesting virtual machine resource limit increases:

* Standard Esv3 Family vCPUs: minimum of 50 per designated region
* Standard Dv4 Family vCPUs: minimum of 10 per designated region

!!! Note
      When you deploy your clusters in HA mode, BigAnimal uses multiple availability zones in the region where you plan to deploy the cluster. Ensure that all regions designated for the deployment of your clusters have no virtual machine SKU restrictions for the Esv3 and Dv4 VM size families in zone 1, 2, and 3.

Reply:
@xugy99 can you review the above text and suggest changes if required?